lib/facility_factory.rb

```diff
@@ -0,0 +1,51 @@
+class FacilityFactory
```

Comment:
I think this is a really great start. One thing to consider is how can we make one `create_facilities(state_data)` method that can be dynamic to read in any state that we pass it in our test?
I like the helpers that you made to format numbers and addresses, can these also be dynamic??